docs/molgenis/run_systemd.md

```diff
@@ -0,0 +1,179 @@
+
+# Install Molgenis emx2 as a system service 
```

Comment:
In overige documentatie wordt MOLGENIS met hoofdletters geschreven. emx2 = EMX2 (al wordt dit in sommige delen van de documentatie ook weggelaten).